src/main/java/taxi/controller/authentication/LogoutController.java

```diff
@@ -0,0 +1,20 @@
+package taxi.controller.authentication;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+
+@WebServlet(urlPatterns = "/logout")
+public class LogoutController extends HttpServlet {
+    @Override
+    public void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        HttpSession session = req.getSession();
+        session.invalidate();
+        resp.sendRedirect(req.getContextPath() + "/index");
```

Comment:
```suggestion
        resp.sendRedirect(req.getContextPath() + "/login");
```
i had same and mentor asked to change to /login